deploy.yml

```
@@ -0,0 +1,72 @@
+- name: Deploy content to static server
+  hosts: all
+  vars:
+    branch_list:
+      - ep2024
+      - ep2025
+    branch_name:
+      "{{ lookup('pipe', 'git rev-parse --abbrev-ref HEAD') | trim }}"
+    sanitized_branch_name:
+      "{{ branch_name | regex_replace('[^a-zA-Z0-9_-]', '_') }}"
+    remote_base_path: "/home/{{ ansible_user }}/content"
+
+  tasks:
+    - name: Generate UUID for temp directory
+      set_fact:
+        unique_id: "{{ lookup('pipe', 'uuidgen') }}"
+
+    - name: Define and trim deploy path
+      set_fact:
+        deploy_path: >-
+          {{
+            (
+              remote_base_path + '/europython_websites/' + branch_name
+              if branch_name in branch_list else
+              remote_base_path + '/previews/' + sanitized_branch_name
+            ) | trim
+          }}
+        temp_dir:
+          "{{ remote_base_path }}/temp/temp_{{ sanitized_branch_name }}_{{
+          unique_id }}"
```

```suggestion
          "{{ remote_base_path }}/temp/{{ unique_id }}"
```
This is the temp directory. Which goes against all unix conventions so it can not be from the system. And the only manual use would be for the rare edge case where copy was successive but the move would not work. In which case `stat` information (creation timestamp) should be enough.

And why do we use this? https://docs.ansible.com/ansible/latest/collections/ansible/posix/synchronize_module.html#parameter-delay_updates is the default ?